### PR TITLE
Add XML prettification

### PR DESCRIPTION
**Pull request by nikolaybaychenko**
* Add optional XML prettification parameter for `->toXml()' method
* Add XML prettification instructions in wiki's Usage section

### CODE REVIEW

File: src/ArrayToXml.php [hunk 1]
```diff
@@ -67,8 +67,13 @@ public static function convert(
         return $converter->toXml();
     }
 
-    public function toXml(): string
+    public function toXml(bool $isPretty = false): string
```
Comment (freekmurze) [inline src/ArrayToXml.php:70]: I like the functionality, but I'd rather have this a separate method called `prettify` that set some instance property so `toXml` will prettify the output.

Comment (nikolaybaychenko) [reply src/ArrayToXml.php:70]: Ok, I can do that instead

**Comment by joshhornby** [comment]
Think this was meant to be merged as part of the 2.12.0 release?

https://github.com/spatie/array-to-xml/blob/master/CHANGELOG.md#2120---2020-06-16

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**